src/main/java/no/nav/finnkandidatapi/midlertidigutilgjengelig/MidlertidigUtilgjengeligService.java

```diff
@@ -44,12 +55,31 @@ public boolean midlertidigTilgjengeligEksisterer(String aktørId) {
             return Optional.empty();
         }
 
-        return repository.hentMidlertidigUtilgjengelig(aktørId);
+        var response = repository.hentMidlertidigUtilgjengelig(aktørId);
+
+        applicationEventPublisher.publishEvent(new MidlertidigUtilgjengeligEndret(
+                MidlertidigUtilgjengelig.builder()
+                        .aktørId(aktørId)
+                        .tilDato(tilDato)
+                        .sistEndretAvIdent(innloggetVeileder.getNavIdent())
+                        .sistEndretAvNavn(innloggetVeileder.getNavn())
+                        .build()));
+
+        return response;
     }
 
-    public Integer slettMidlertidigUtilgjengelig(String aktørId) {
-        return repository.slettMidlertidigUtilgjengelig(
+    public Integer slettMidlertidigUtilgjengelig(String aktørId, Veileder innloggetVeileder) {
+        var response = repository.slettMidlertidigUtilgjengelig(
                 aktørId
         );
+        applicationEventPublisher.publishEvent(
+                new MidlertidigUtilgjengeligSlettet(MidlertidigUtilgjengelig.builder()
+                        .aktørId(aktørId)
+                        .sistEndretAvIdent(innloggetVeileder.getNavIdent())
+                        .sistEndretAvNavn(innloggetVeileder.getNavn())
+                        .build())
```

Comment:
Mangler det å sette endel felter her i `MidlertidigUtilgjengelig`?
Hverken `tilDato` eller `fraDato` er satt...

Reply:
Ser at i de andre servicene som publiserer events så henter vi opp objektet som ligger i eventet fra databasen. Hva er grunnen for at vi ikke gjør det samme her?

Reply:
Ah ja den finnes jo ikke i databasen lengre siden den er slettet.
Trenger vi da ha med hele `MidlertidigUtilgjengelig` objektet i `MidlertidigUtilgjengeligSlettet` eventen?
Kunne kanskje sendt med eventet kun de feltene som vi trenger for å sende en "slett-midlertidig-utilgjengelig-melding" til Kafka?